types/koa-cash/koa-cash-tests.ts

```diff
@@ -7,7 +7,7 @@ app.use(koaCash());
 app.use(async ctx => {
     // this response is already cashed if `true` is returned,
     // so this middleware will automatically serve this response from cache
-    if (await ctx.cashed()) return;
+    if (await ctx.cashed()) return ctx.cashClear('/');
```

Comment:
Wasn't sure at first, but the latest build error confirms it. `cashClear()` returns `void`, so this doesn't make sense. Maybe something like this:

```suggestion
    if (await ctx.cashed()) {
      ctx.cashClear('/');
      return;
    }
```

Wouldn't make much sense for an application, but whatever :smile: